core/src/main/resources/hudson/model/Messages.properties

```diff
@@ -276,20 +276,33 @@ UpdateCenter.Status.ConnectionFailed=\
   <span class=error>Failed to connect to {0}. \
   Perhaps you need to <a href="../pluginManager/advanced">configure HTTP proxy?</a></span>
 UpdateCenter.init=Initialing update center
+UpdateCenter.PluginCategory.android=Android Development
 UpdateCenter.PluginCategory.builder=Build Tools
 UpdateCenter.PluginCategory.buildwrapper=Build Wrappers
 UpdateCenter.PluginCategory.cli=Command Line Interface
 UpdateCenter.PluginCategory.cluster=Cluster Management and Distributed Build
+UpdateCenter.PluginCategory.database=Database
+UpdateCenter.PluginCategory.deployment=Deployment
+UpdateCenter.PluginCategory.devops=DevOps
+UpdateCenter.PluginCategory.dotnet=.NET Development
 UpdateCenter.PluginCategory.external=External Site/Tool Integrations
+UpdateCenter.PluginCategory.groovy-related=Groovy-related
+UpdateCenter.PluginCategory.ios=iOS Development
+UpdateCenter.PluginCategory.library=Library plugins (for use by other plugins)
 UpdateCenter.PluginCategory.listview-column=List view columns
 UpdateCenter.PluginCategory.maven=Maven
 UpdateCenter.PluginCategory.misc=Miscellaneous
 UpdateCenter.PluginCategory.notifier=Build Notifiers
 UpdateCenter.PluginCategory.page-decorator=Page Decorators
+UpdateCenter.PluginCategory.parameter=Build Parameters
 UpdateCenter.PluginCategory.post-build=Other Post-Build Actions
+UpdateCenter.PluginCategory.publisher=Other Post-Build Actions
```

Comment:
I doubt it makes much sense to distinguish builders and publishers, because Flexible Publish, AnyBuildStep, Workflow and other such guys remove the difference even now.. One generic category should be better.